### PR TITLE
Backwards compatibility fixes

### PR DESCRIPTION
**Pull request by iulianpascalau**
- removed unnecessary backwards compatibility flag from stakingToPeer as the v1.1.10 does not change the peer state on unBond operation.

### CODE REVIEW

File: vm/systemSmartContracts/validator.go [hunk 1]
```diff
@@ -1263,6 +1263,12 @@ func (v *validatorSC) unBondNodesFromStakingSC(blsKeys [][]byte) [][]byte {
 }
 
 func (v *validatorSC) unBondV1(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if len(args.Arguments) > 0 {
```
Comment (sasurobert) [inline vm/systemSmartContracts/validator.go:1266]: delete this

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: vm/systemSmartContracts/validator.go (hunk 1 above)
Comment (iulianpascalau) [reply vm/systemSmartContracts/validator.go:1266]: :man_facepalming:

Comment (iulianpascalau) [reply vm/systemSmartContracts/validator.go]: done

**Review by iulianpascalau — COMMENTED** [review]
Looks good :+1:

**Review by sasurobert — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by mateigraura — APPROVED** [review]
system test passed

**Pull request merged**